infrastructure/cdn-in-a-box/readiness/run.sh

```diff
@@ -41,7 +41,7 @@ while true; do
 
     success="true"
     for u in "${exampleURLs[@]}"; do
-        status=$(curl -Lkvs --connect-timeout 2 -m5 -o /dev/null -w "%{http_code}" "$u")
+        status=$(curl -Lkvs --connect-timeout 2 -m5 -o /dev/null -w "%{http_code}" "$u" || true)
```

Comment:
To avoid the `|| true`, this can be
```bash
if ! status=$(curl -Lkvs --connect-timeout 2 -m5 -o /dev/null -w "%{http_code}" "$u") || [[ "$status" -ne 200 ]]; then
```